frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,65 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  margin: "0px",
+  width: "100%",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+  fontSize: "24px",
+});
+
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
```

Comment:
I think these are 40px in figma

Reply:
ah, just noticed the height isn't consistent across the mocks. Left a comment for Kanchan to confirm.